rust/main/submitter/src/chain_tx_adapter/chains/sealevel/adapter.rs

```diff
@@ -292,43 +298,26 @@ impl AdaptsChain for SealevelTxAdapter {
 
         info!(?tx, "confirmed transaction by signature status");
 
-        let executed = self
-            .submitter
-            .confirm_transaction(signature, CommitmentConfig::processed())
-            .await
-            .map_err(|err| {
-                warn!(
-                    "Failed to confirm process transaction with commitment level processed: {}",
-                    err
-                )
-            })
-            .unwrap_or(false);
-
-        info!(?tx, "confirmed transaction with commitment level processed");
-
-        if !executed {
-            return Err(SubmitterError::TxSubmissionError(
-                "Process transaction is not confirmed with commitment level processed".to_string(),
```

Comment:
this step would always return an error in e2e, causing txs to be dropped. It's better to rely on querying the status via the txhash imo